src/core/Akka/Actor/ActorRef.cs

```diff
@@ -66,6 +66,7 @@ public FutureActorRef(TaskCompletionSource<object> result, Action unregister, Ac
             _result = result;
             _unregister = unregister;
             _path = path;
+            _result.Task.ContinueWith(_ => _unregister());
```

Comment:
These changes also appear to have fixed https://github.com/akkadotnet/akka.net/issues/1448 as well. Could have also been the previous PR, but I don't see that message appear in the logs for NBench, unit tests, or multi-node tests (spot checked a bunch of them.)